yandex_contest/monitoring.py

```diff
@@ -0,0 +1,19 @@
+def main():
+    rows_number = int(input())
```

Comment:
[tiny] I think `number` is not the best name here, `count` express the intent better. And similar with `columns_` variable.
```suggestion
    rows_count = int(input())
```